css/layout.css

```diff
@@ -0,0 +1,106 @@
+main {
+  width: 480px;
+  margin: 0 auto;
+}
+
+div {
+  width: 480px; 
+  margin: 0 auto; 
+}
+
+footer {
+  width: 480px;
+  height: 80px;
+  /* float: right; */
+  margin: 0 auto;
+}
+
+#a {
+  width: 480px;
+  height: 75px;
+
+}
+
+nav {
+  width: 480px;
+  margin: 0 auto;
+  height: 150px;
+}
+
+#c, #d {
+  width: 480px;
+  height: 125px;
+}
+
+.twoColContainer {
+  width: 480px;
+
+}
+#e, #f, #g, #h {
+  width: 240px;
+  height: 250px;
+}
+
+#e, #g {
+  float: left;
+}
+#f, #h {
+  float: right;
+}
+
+#i {
+  display: inline-block;
+  width: 480px;
+  height: 80px;
+  margin: 0 auto;
+}
+
+@media only screen and (min-width:960px){
+  main {
+    width: 960px;
+  }
+  div {
+    width: 960px;
+    margin: 0 auto;
+  }
+  #a {
+    width: 960px;
+    height: 100px;
+    line-height: 100px;
+  }
+  nav {
+    float: left;
+    display: block;
+    margin: 0 auto;
+    width: 200px;
+    height: 600px;
+  }
+  #c, #d {
+    width: 380px;
+    height: 200px;
+  }
+  #c {
+    float: left;
+  }
+  #d {
+    float: left;
+  }
+  #e, #f, #g, #h {
```

Comment:
Since you ended up writing this sort of declaration twice, I'll point out that you also could have created a class that applied to these four elements (like `.columns`). It's personal preference on which is cleaner. (I come from the Google style guide, which literally only allows class-based CSS, so that's my bias.)